src/lwc/gsApplicationStatus/gsApplicationStatus.html

```diff
@@ -0,0 +1,37 @@
+<template >
+    <template if:false={isActiveInstance}>
+        <lightning-card>
+            <div class="spinner" if:true={isLoading}>
```

Comment:
Better to use a `<template>` tag to wrap this in for visibility than to use `if:...` in the div tag - throughout this and all component html

Reply:
I think you should be able to put the `if:true` directly into the `lightning-spinner` tag.